miden-lib/asm/kernels/transaction/lib/account.masm

```diff
@@ -144,29 +146,32 @@ const.ACCOUNT_PUSH_PROCEDURE_INDEX_EVENT=131082
 #! Fungible faucet: The faucet data consists of [0, 0, 0, total_issuance]
 #! Non-fungible faucet: The faucet data consists of SMT root containing minted non-fungible assets.
 #!
-#! Stack: []
-#! Output: [faucet_storage_data_slot]
+#! Inputs:  []
+#! Outputs: [faucet_storage_data_slot]
```

Comment:
I think it is a convention matter. Almost all procedures which should be `exec`uted work in an assumption that there are other elements on the stack, so it can't modify any elements other than those specified in the inputs. 

My intention was to remove ellipsis from the inputs/outputs description because we use it everywhere, so we can just set the convention that we always have elements deeper on the stack, so we can remove excess ellipsis.

But I'm fine if we add ellipsis to the description, I'll do as you say.

Reply:
I do think using ellipsis would be a bit cleaner - but this is a pretty minor point and I'm fine w/o them too.